packages/perseus/src/widgets/categorizer/categorizer.util.test.ts

```diff
@@ -0,0 +1,32 @@
+import getCategorizerPublicOptions from "./categorizer.util";
+
+import type {PerseusCategorizerWidgetOptions} from "../../perseus-types";
+
+describe("getCategorizerPublicOptions", () => {
```

Comment:
Would love to add more tests if anyone can think of other useful things to check :)